docs/developer/vagrant.rst

```diff
@@ -11,6 +11,9 @@ get started, simply use these commands::
     $ sudo systemctl enable libvirtd
     $ sudo systemctl start libvirtd
 
+If you're on Ubuntu, use this in place of the first command::
+		$ sudo apt install ansible libvirt-dev vagrant-libvirt vagrant-sshfs
```

Comment:
The indentation seems a little bit off, I think that it should be 4 spaces to look like the rest of the document